src/components/Navbar.tsx

```diff
@@ -8,15 +11,24 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+          <NavLink
+            className={({ isActive }) => cn('navbar-item', {
+              'has-background-grey-lighter': isActive,
+            })}
+            to="/"
+          >
+            Home
+          </NavLink>
 
-          <a
+          <NavLink
             aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+            className={({ isActive }) => cn('navbar-item', {
+              'has-background-grey-lighter': isActive,
+            })}
+            to="/people"
```

Comment:
From the task description

> Keep search params when navigating within the People page (when selecting a person or clicking the People link).

Tip: Here, you can get the current `searchParams` and pass the object with the corresponding `pathname` and `search` properties.